### PR TITLE
Migrate GetSessionResponseStatsAction

### PR DESCRIPTION
**Pull request by daongochieu2810**
Part of https://github.com/TEAMMATES/teammates/issues/12048

Outline of Solution

Modified GetSessionResponseStatsAction to work with dual DB.

### CODE REVIEW

**Comment by daongochieu2810** [comment]
This action has some functions that https://github.com/TEAMMATES/teammates/pull/12258 implemented, so this can wait for that one to be merged first

**Comment by cedricongjh** [comment]
hi @daongochieu2810, just a reminder to fix the linting! (or re-run the tests if its already fixed)

File: src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java [hunk 1]
```diff
@@ -182,6 +187,48 @@ public FeedbackSession publishFeedbackSession(String feedbackSessionName, String
...
+            expectedTotal += usersLogic.getStudentsForCourse(fs.getCourse().getId()).size();
+        }
+
+        // Pre-flight check to ensure there are questions for instructors.
+        if (!fqLogic.hasFeedbackQuestionsForInstructors(fs.getFeedbackQuestions(), true)) {
+            return expectedTotal;
+        }
+
+        List<String> instructorEmails = usersLogic.getInstructorsForCourse(
+                fs.getCourse().getId()).stream().map(User::getEmail).collect(Collectors.toList());
```
Comment (samuelfangjw) [inline src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java:206]: I don't think there's a need to map to instructorEmails. Would it be better to change this to a list of instructors instead?

**Comment by kevin9foong** [comment]
hi @cedricongjh, checking in if this PR can be merged

**Comment by cedricongjh** [comment]
replaced by: https://github.com/TEAMMATES/teammates/pull/12753

**Pull request closed without merging**